pyblish/lib.py

```diff
@@ -57,8 +57,18 @@ def extract_traceback(exception):
     """Inject current traceback and store in exception"""
     exc_type, exc_value, exc_traceback = sys.exc_info()
     exception.traceback = traceback.extract_tb(exc_traceback)[-1]
```

Comment:
About the new `error_info` key, I think these are already part of what you can get out of the `error` key.

```py
>>> from pyblish import api
>>> class Plugin(api.ContextPlugin):
...   def process(self, context):
...     assert False, "This was a problem"
...
>>> api.register_plugin(Plugin)
>>> ctx = util.publish()
This was a problem
>>> error = ctx.data["results"][-1]["error"]
>>> error
AssertionError('This was a problem')
>>> error.traceback
<FrameSummary file <stdin>, line 3 in process>
>>> dir(error.traceback)
['__class__', '__delattr__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__iter__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__slots__', '__str__', '__subclasshook__', '_line', 'filename', 'line', 'lineno', 'locals', 'name']
>>> tb = error.traceback
>>> tb.lineno
3
>>> tb.name
'process'
>>> tb.filename
'<stdin>'
```

Reply:
we'll certainly double check that maybe just missed it

Reply:
not sure if it was my fault but in pyblish-lite the error.traceback still was traceback object but all attributes returned `<string>` which was in GUI showed as an empty string

Reply:
Oh, my message disappeared... Well again:
I've investigated a little bit and found out that it was issue in Nuke and Maya:
- With pyblish-lite in Nuke or Maya `error.traceback` is not `FrameSummary` but `tuple` object. Values can be extracted with `fname, lineno, func, exc = error.traceback` but `fname` has value `"<string>"` and `exc` is `"None"` (lineno and func store values as expected)
- This issue won't happen when mocked pyblish-lite is launched -> I assume that pyblish-lite uses Nuke's or Maya's `traceback` module which has modified implementation of `extract_tb` ? (now I noticed it makes no sence because pyblish-base's method `extract_traceback` happens in same thread as pyblish-lite with the difference that there extraction works)

BTW: is possible to extract stack from error.traceback?